tests/unit-tests/test-class-utils.php

```diff
@@ -382,7 +382,7 @@ public function testGetTargetResumeId_WhenCalled_ReturnsLastLessonIdIfNotComplet
 		$this->assertEquals( $course_lessons['lesson_ids'][0], $result );
 	}
 
-	public function testGetTargetResumeId_WhenCalled_ReturnsCourseIdIfNoLessonIsCompleted() {
+	public function testGetTargetPagePostIdForContinueUrl_WhenCalled_ReturnsFirstLessonIdIfNoLessonIsCompleted() {
```

Comment:
```suggestion
	public function testGetTargetPagePostIdForContinueUrl_WhenNoLessonIsCompleted_ReturnsFirstLessonId() {
```
I want to highlight and explain this suggestion.
We put condition in the second part of the name, and result in the last part of the name.
`WhenCalled` and `Always` refer to the same idea – code has the same result no matter what happens. In this case there is an exact condition, but it was in the very end, so the name of the test could lead to false assumptions.

Reply:
Oh, good point, thanks!